app/views/support_interface/support_users/index.html.erb

```diff
@@ -37,6 +40,14 @@
         <td class='govuk-table__cell'>
           <%= support_user.dfe_sign_in_uid %>
         </td>
+        <% if FeatureFlag.active?('delete_support_users') %>
+          <td class='govuk-table__cell'>
+            <%= link_to 'Delete',
```

Comment:
I think this should just be a link, "Delete user", rather than a button.

Reply:
This would benefit from some hidden text such as "Delete user <span class="govuk-visually-hidden">foo@example.com</span>" for accessibility 👍

Reply:
https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1469/files#diff-e231e5182cc5a11d49224019f20a33a8R46